tests/test_utils.py

```diff
@@ -47,6 +47,25 @@ def test_lazy_load_subclass(lit_llama):
                 torch.testing.assert_close(actual._load_tensor(), expected)
 
 
+def test_incremental_write(lit_llama):
+    import lit_llama.utils
+
+    sd = {str(k): torch.randn(5, 10) for k in range(3)}
+    sd_expected = {k: v.clone() for k, v in sd.items()}
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        path = pathlib.Path(tmpdirname)
```

Comment:
These 2 lines can be replaced with the `tmp_path` pytest fixture

Reply:
Thank you!